crates/kitsune_p2p/types/src/config.rs

```diff
@@ -138,7 +138,7 @@ pub mod tuning_params_struct {
 
         /// The max number of bytes of op data to send in a single message.
         /// Payloads larger than this are split into multiple batches.
```

Comment:
Nothing to do with your PR but noticing that this could be helpful.

```suggestion
        /// Payloads larger than this are split into multiple batches
        /// when possible (currently, a single Op exceeding this size
        /// will not be further split up, and there are other cases where
        /// densely populated DHT regions may contain more than this
        /// limit when transferred.
```